src/openforms/config/migrations/0059_alter_globalconfiguration_form_upload_default_file_types.py

```diff
@@ -4,6 +4,35 @@
 import django_jsonform.models.fields
 
 
+def add_extra_zip_mimetypes(apps, _):
+    """
+    Set up the correct zip mimetypes.
+
+    This ensures all the allowed mimetypes concerning zip files are included.
+    """
+    GlobalConfiguration = apps.get_model("config", "GlobalConfiguration")
+    if not GlobalConfiguration.objects.exists():
+        return
+
+    config = GlobalConfiguration.objects.get()
+    if current_mimetypes := config.form_upload_default_file_types:
+        needed_mime_types = (
+            "application/x-zip-compressed",
+            "application/zip-compressed",
+        )
+
+        def add_if_missing(current_list: list[str]):
+            for item in needed_mime_types:
+                if item not in current_list:
+                    current_list.append(item)
+
+        if "application/zip" in current_mimetypes:
+            add_if_missing(current_mimetypes)
```

Comment:
this isn't right - the global config is an arrayfield in which each item has a possible choice value - the correct choice value is `UploadChoices.zip` (which is a single string with comma-separated mime types). In this patch, it looks like you're adding individual items to the arrayfield.

I believe I have the correct patch in my PR: https://github.com/open-formulieren/open-forms/pull/4703/files#diff-c3b529919eccd3ced5e9d359d132192403872ba6355a07979000ee7597b3d4e3R21-R25